lib/ws-kernel-picker.js

```diff
@@ -91,7 +97,9 @@ export default class WSKernelPicker {
           this.onSession.bind(this)
         );
         sessionListing.previouslyFocusedElement = this.previouslyFocusedElement;
-        sessionListing.setLoading("Loading sessions...");
+        sessionListing.selectListView.update({
+          loadingMessage: "Loading sessions..."
+        });
```

Comment:
I think the code only shows up once Hydrogen is activated (e.g. once you run some code).